### ISSUE TITLE
Dev Console permanent

### ISSUE THREAD

**Issue by TallTacoTristan** [BrowserWorks/waterfox#3536]
### What happened?

I was testing code using browser console, when things like mutationobserservers or libraries are added, they stay, even if you reload the page, or clear your browser data completely

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

G6

### What platform are you seeing the problem on?

Windows

**Comment by MrAlex94** [comment]
Do you mean that in the dev console when you execute something such as:
`let one = 1;` that even if you refresh the page/clear browser data, `one` remains defined? Or am I misunderstanding?

**Comment by TallTacoTristan** [comment]
Yes, While not all commands and sites do it, while on https://app-registro.imssbienestar.gob.mx/ i pasted my userscript in to test code without having to re-complete a captcha and the scripts functionality continued to occur until i switched browsers completely, i've since deleted the userscript so i cant reproduce it rn but specifically i was using jspdf and mutationobservers and i've never had issues like this on firefox
